hail/python/hailtop/hailctl/dataproc/connect.py

```diff
@@ -93,12 +95,13 @@ async def main(args, pass_through_args):  # pylint: disable=unused-argument
         chrome = os.environ.get('HAILCTL_CHROME') or get_chrome_path()
 
         # open Chrome with SOCKS proxy configuration
-        with open(os.devnull, 'w') as f:
-            subprocess.Popen([
-                chrome,
-                'http://localhost:{}'.format(connect_port_and_path),
-                '--proxy-server=socks5://localhost:{}'.format(args.port),
-                '--host-resolver-rules=MAP * 0.0.0.0 , EXCLUDE localhost',
-                '--proxy-bypass-list=<-loopback>',  # https://chromium.googlesource.com/chromium/src/+/da790f920bbc169a6805a4fb83b4c2ab09532d91
-                '--user-data-dir={}'.format(tempfile.gettempdir())
-            ], stdout=f, stderr=f)
+        subprocess.Popen([
+            chrome,
+            'http://localhost:{}'.format(connect_port_and_path),
+            '--proxy-server=socks5://localhost:{}'.format(args.port),
+            '--host-resolver-rules=MAP * 0.0.0.0 , EXCLUDE localhost',
+            '--proxy-bypass-list=<-loopback>',  # https://chromium.googlesource.com/chromium/src/+/da790f920bbc169a6805a4fb83b4c2ab09532d91
+            '--user-data-dir={}'.format(
+                os.path.join(tempfile.gettempdir(),
+                             'hailctl-dataproc-connect-' + secret_alnum_string(6)))
+        ], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```

Comment:
this is a special value that indicates that `os.devnull` should be used, added in python 3.3